templates/base.jinja2

```diff
@@ -35,6 +35,7 @@
       </nav>
     </header>
     <main>
+      <script src="//code.jquery.com/jquery-1.11.2.min.js"></script>
```

Comment:
script tags belong in the `<head>` of the page, not the `<body>`.  Especially those that load remote resources like the JQuery library.